data/virtualization/autoyast/guest_15.xml.ep

```diff
@@ -62,6 +62,13 @@
       </listentry>
      % }
     % }
+    % unless ($check_var->('TERADATA', "")) {
```

Comment:
Please add one more check if the version in $vm_name is equal to the value of "TERADATA". Otherwise it adds the TD repo to all guests.